pylib/gyp/input.py

```diff
@@ -2536,6 +2536,8 @@ def ProcessListFiltersInDict(name, the_dict):
     lists = []
     del_lists = []
     for key, value in the_dict.items():
+        if key == "":
```

Comment:
```suggestion
        if not key:
```
All falsey dict keys are just an unwanted as the empty string.
`d = {"": 0, None: 1, False: 2, (): 3, frozenset(): 4, 0: 5, 0.0: 6}`